jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/MetaInfConfiguration.java

```diff
@@ -720,25 +718,21 @@ public FileVisitResult visitFile(Path f, BasicFileAttributes attrs) throws IOExc
      */
     public Collection<URL> getTlds(URI uri) throws IOException
     {
-        HashSet<URL> tlds = new HashSet<URL>();
-
+        HashSet<URL> tlds = new HashSet<>();
         URI jarUri = uriJarPrefix(uri, "!/");
```

Comment:
The `MultiReleaseJarFile` PR has a fix for this as well.
`URI Resource.toJarURI(URI)